.github/workflows/make-release.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021-2024 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+name: "Make release"
+
+on:
+  push:
+    tags:
+      - "openssl-*"
+  workflow_dispatch:
```

Comment:
Doesn't this allow running this workflows manually at any commit? How does that interact with `github.ref_name`?

Reply:
Yes, it manually runs the workflow on any commit. If a commit has a tag the name of the tag will be taken. Main use case of this trigger is a backup when for some reason the workflow run fails on a tag push. Then you don't need to push a new tag to run the workflow again.

Reply:
I still want to know what happens if the top commit of the repository isn't tagged.  I see the possibility of screwups, so perhaps there needs to be some condition in the steps to avoid doing the work in such a case?

Reply:
Let me just remove it.